File: python/7 kyu/360.py
# https://www.codewars.com/kata/54b45c37041df0caf800020f/train/python

# Passed

from math import gcd

def binary_gcd(x, y):
    value = gcd(x, y)
    value_binary = bin(value)
    result = value_binary.split("b")[1].count('1')
    return result

output = binary_gcd(666666, 333111)
print(output)

File: nodejs/8 kyu/360.js
// https://www.codewars.com/kata/55a2d7ebe362935a210000b2/train/javascript

// Passed

function findSmallestInt(arr) {
  let result = Math.min(...arr);
  return result;
}

const output = findSmallestInt([78, 56, 232, 12, 8]);
console.log(output);